source/PlayState.hx

```diff
@@ -39,12 +43,22 @@ class PlayState extends FlxState
 			1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],
 			20, 15, AssetPaths.tiles__png, 32, 32);
 		add(map);
+		
+		//make-a da blockies
```

Comment:
This is debug code and probably shouldn't be pulled into master. Really, the level should have these objects stored in it and create them when it is loaded, and not just be defined in the playstate.